public/static/docs/glossary.js

```diff
@@ -37,6 +37,14 @@ contain \`.git/\` and [\`.dvc/\`](/doc/user-guide/dvc-files-and-directories)
 directories, as well as any DVC-files created by DVC.
       `
     },
+    {
+      name: 'Git repository',
```

Comment:
it looks to me we are overcomplicating this cc @jorgeorpinel ... no need to expain Git repo just for the sake of mentioning that it can be a DVC repo. Let's find some other explicit way in the text of doing this

Reply:
Agree. Let's not add this term to the glossary. The glossary is for basic DVC concepts. Refer to #461 and #542.

Plus we already have the terms "DVC project" (mentions that it can be a Git repo) and "DVC repository".

Reply:
Removed

Reply:
Thanks for the initiative though.